modules/openapi/src/alloy/openapi/package.scala

```diff
@@ -32,24 +31,10 @@ package object openapi {
 
   def convertWithConfig(
       model: Model,
-      allowedNS: Option[Set[String]],
+      service: Shape,
```

Comment:
Only thought I have is this looks like we could just take the ShapeId rather than the full shape. Otherwise looks good to me

Reply:
I thought about it, but:

- we need to get the traits of the shape, so we'll eventually need to look it up in the model. If the user already has the shape, that's just wasteful (shape -> shapeid -> model.expectShape)
- if the user only has a ShapeId, they can do the lookup themselves

the waste is minimal either way, I guess, so it's a bit of a coin flilp to me...

Reply:
and I don't want to add _another_ overload 😅

Reply:
Ahh yeah I missed that, thanks for explaining 🙏